CRM/Odoosync/Sync/Contribution/Data/Payment.php

```diff
@@ -63,6 +63,8 @@ private function mappedItems($paymentItemsDao) {
     $paymentItems = [];
 
     while ($paymentItemsDao->fetch()) {
+      $paymentDate = CRM_Odoosync_Common_Date::convertDateToTimestamp($paymentItemsDao->payment_date);
```

Comment:
maybe it is better to change : 

convertDateToTimestamp() 

method to to the conversion internally ?